packages/contracts-sdk/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@snickerdoodlelabs/contracts-sdk",
-  "version": "4.0.8",
+  "version": "4.0.13",
```

Comment:
why is the version number jumping up so much here? Do we have other contract updates on another branch?